src/nrniv/CMakeLists.txt

```diff
@@ -354,3 +354,23 @@ install(TARGETS nrniv_lib DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
 if (LIBIVX11DYNAM_NAME)
   install(FILES ${PROJECT_BINARY_DIR}/lib/${LIBIVX11DYNAM_NAME} DESTINATION lib)
 endif()
+
+# =============================================================================
+# Install / copy cpp and required headers for binary special flavor
+# =============================================================================
+# ~~~
+# Note: we also added following headers in NeuronFleLists.cmake:
+#   ivstream.h nrnconf.h nrnmpi.h nrnmpidec.h
+# ~~~
+
+# For testneuron
```

Comment:
`nitpick` : this comment is not cleat to me. I assume this is always required for building special.

Reply:
Well it's for when we have `-DNRN_ENABLE_TESTS=ON` (`testneuron` in CTest). 
Because these files are used afterwards when you do `nrnivmodl`, so normally it would have been just for install.